RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforDevice.h

```diff
@@ -122,6 +124,53 @@ namespace pixelCPEforDevice {
     cotbeta = gvy * gvz;
   }
 
+  constexpr inline float check_for_short(uint16_t upper_edge_first_pix,  //!< As the name says.
```

Comment:
could you rename this to `checkForShort`, to match the convention used in the other functions in this file ?

Reply:
or maybe, `updatePositionForShortCluster` or some other meaningful name ?